modules/ROOT/pages/design-storage-github.adoc

```diff
@@ -0,0 +1,99 @@
+= Storing Projects in GitHub
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:imagesdir: ../assets/images
+
+By default, projects that you create in API Designer are saved in Design Center. However, you can save a project as a repository in GitHub.
```

Comment:
Should we make clear in the previous text that although you have these two options, the default is Design Center?